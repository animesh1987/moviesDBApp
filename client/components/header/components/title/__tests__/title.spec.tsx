import * as React from 'react';
import { create } from 'react-test-renderer';

import { Title } from '../index';

describe('Title Test', () => {

  it('renders correctly', () => {
    const component = create(<Title />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});