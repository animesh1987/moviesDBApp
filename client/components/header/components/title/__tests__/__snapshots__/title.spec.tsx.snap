// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Test renders correctly 1`] = `
<h3>
  netflixroulette
</h3>
`;
